### ISSUE TITLE
Wrong Password no way to recover

### ISSUE THREAD

**Issue by timot59** [Berry-Pool/nami-wallet#411]
Hello,

So i tried to connect cnft.io to my nami wallet, to  this day it was good, then now it says my password is not good while its written down on paper and on a secret file on my computer, 100% sure it is the good password , there is no way for me to do anything, i can not reset the acccount , I have my seed phrase, i dont understand. PLease help me

**Comment by Novacastrius69** [comment]
yep this is happenin often that and importing wallets onto new devices
